scripts/check_frontend_coverage_test.py

```diff
@@ -180,7 +176,8 @@ def mock_exists(unused_path):
         with exists_swap:
             with self.assertRaises(Exception) as error:
                 check_frontend_coverage.check_coverage_changes()
-            self.assertTrue(error.exception.message)
+            self.assertRaisesRegexp(Exception, 'Expected lcov file to be'
```

Comment:
Hm, is this code executed? I think the idea is to replace line 177 with this (see how assertRaisesRegexp is used elsewhere in the codebase).

Reply:
Done.